models/mailchimp/mailchimp_campaigns.sql

```diff
@@ -0,0 +1,3 @@
+select distinct
```

Comment:
yes, the idea was to make sure there are no duplicates in the data because any subsequent joins would create problems. We could leave it up to the person working with these models to perform these tests if you think that's more reasonable.